packages/pluggableWidgets/radio-buttons-native/src/RadioButtons.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.native.radiobuttons.RadioButtons"
+        pluginWidget="true"
+        needsEntityContext="true"
+        offlineCapable="true"
+        supportedPlatform="Native"
+        xmlns="http://www.mendix.com/widget/1.0/"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../node_modules/mendix/custom_widget.xsd">
+    <name>Radio Buttons</name>
+    <description/>
+    <studioProCategory>Input elements</studioProCategory>
+    <studioCategory>Input elements</studioCategory>
+    <icon/>
+    <properties>
+    <propertyGroup caption='General'>
+        <propertyGroup caption="Data Source">
+            <property key="enum" type="attribute" >
+                    <caption>Select an enum</caption>
+                    <description>Select radio button options</description>
+                    <attributeTypes>
+                        <attributeType name="Enum"/>
+                    </attributeTypes>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Formatting">
+                <property key="orientation" type="enumeration" defaultValue="vertical">
+                    <caption>Orientation</caption>
+                    <description>Options orientation</description>
```

Comment:
This could explain whether the group is stacked (vertical) or in a row. Chris, what do you think?

Reply:
Again i don't think a description is necessary here. Its pretty self explanatory

Reply:
Move the property to the "General" group of the widget